### ISSUE TITLE
Optional RegML regLetter element treated as required

### ISSUE THREAD

**Issue by chosak** [cfpb/eregs-2.0#50]
The RegML schema includes [an optional `regLetter` element](https://github.com/cfpb/regulations-schema/blob/cfe2f07a9fcda10ff5851eeac73d6d458f85f636/src/preamble.xsd#L22) which provides the letter of the regulation. This is defined as optional in the schema and is absent from many of the RegML files in [regulations-xml](https://github.com/cfpb/regulations-xml), for example all of Reg Z.

The `Preamble` model defines a [`reg_letter`](https://github.com/cfpb/eregs-2.0/blob/a4a7a5fd51e08eb02b46755855797c80b9e01a43/eregs_core/models.py#L321) property that assumes this is always defined. This property is then used in various templates ([for example](https://github.com/cfpb/eregs-2.0/blob/f5f03f8d85c368ba904ee2ec2e6b0e8fdac7b076/eregs_core/templates/main.html#L40)), which means that if you load RegML without that element, it breaks the site.

If this element is optional then the logic to display the letter should use the `section` tag as a fallback.

## Current behavior

RegML files without `regLetter` can't be viewed in this app.

## Expected behavior

RegML files without `regLetter` can be viewed in this app.

## Steps to replicate behavior (include URLs)

1. Load a RegML file without `regLetter`:

```sh
./manage.py import_xml ../regulations-xml/regulation/1013/2016-28710.xml
```

2. Visit http://localhost:8000 and you'll get a `AttributeError at /, 'NoneType' object has no attribute 'text'` due to the missing element.

**Comment by grapesmoker** [comment]
I've been wondering if we should fix this in the schema or not. I'm a little conflicted about it because theoretically there could be regulations that don't have letters (and for other agencies there certainly no letters), but I'm pretty sure that for our purposes we always have a regulation letter. I'm thinking that if we decide to change this in the schema, we should probably tell anyone who wants to use our schema/project that they should change the `reg_letter` property and anything that references it.

**Comment by chosak** [comment]
Is one of the goals of RegML to be applicable to other agencies besides CFPB?

Is the `reg_letter` property just used for display? If so, would it be possible to just have the frontend have some fallback if that letter is defined in the RegML?

**Comment by grapesmoker** [comment]
It's a semi-explicit goal. I mean, 18F is already doing that, so clearly it's being achieved in some ways, but we've never gone out and marketed it to other agencies ourselves, even though maybe we should. It's just a display functionality, so I don't think we'd lose anything if we had a fallback (to what?).

**Comment by grapesmoker** [comment]
I think in the absence of a `regLetter` I'm just going to fall back to `part`, since that's really the key piece of information that `regLetter` maps on to.

**Comment by chosak** [comment]
This was fixed in #56.
